projects/igniteui-angular/src/lib/snackbar/snackbar.component.html

```diff
@@ -1,10 +1,9 @@
-<div class="igx-snackbar" *ngIf="isVisible" (@slideInOut.start)="snackbarAnimationStarted($event)" (@slideInOut.done)="snackbarAnimationDone($event)"
-    [@slideInOut]="isVisible">
-    <div class="igx-snackbar__message" [@fadeInOut]="isVisible">
+<div class="igx-snackbar">
```

Comment:
We no longer need this. Just move the class to the host element using HostBinding.